packages/odyssey-storybook/src/components/odyssey-mui/MenuButton/MenuButton.stories.tsx

```diff
@@ -306,6 +336,38 @@ export const IconButton: StoryObj<MenuButtonProps> = {
     ],
     tooltipText: "More actions",
   },
+  play: async ({ canvasElement, step }: PlaywrightProps<MenuButtonProps>) => {
+    await step("MenuButton Aria-Label", async () => {
+      const canvas = within(canvasElement);
+      const menuButton = canvas.getByLabelText("More actions");
+      expect(menuButton).toHaveAttribute("type", "button");
+    });
```

Comment:
Instead of checking that something is a button, we can instead try to select a button. The `getBy` function will throw an error, which may be what we want, but it's hard to debug.

Instead, we can use `queryBy` to return `null` when the selection fails. In that case, we can write our `expect` to handle that use case accordingly:

```suggestion
    await step("MenuButton Aria-Label", async () => {
      const canvas = within(canvasElement);
      const menuButton = canvas.queryByRole("button", { name: "More actions" });
      expect(menuButton).not.toBeNull();
    });
```